src/main/java/net/dv8tion/jda/api/entities/Role.java

```diff
@@ -271,4 +285,33 @@ default RoleAction createCopy()
      */
     @Nonnull
     JDA getJDA();
+
+
+    /**
+     * Enum used to differentiate between the different types of Guild Roles.
+     */
+    enum Type
```

Comment:
Should either be named RoleType (what current naming + getter suggest; imo preferred), or ManagementType

Reply:
Changed back to RoleType